frontend/src/lib/lemon-ui/LemonInputSelect/LemonInputSelect.tsx

```diff
@@ -400,7 +464,7 @@ export function LemonInputSelect({
                                         ) : undefined
                                     }
                                     sideAction={
-                                        !option.__isInput
+                                        !option.__isInput && allowCustomValues
```

Comment:
This is an unrelated fix - without it, default options could get displayed with the edit side action